Makefile

```diff
@@ -0,0 +1,12 @@
+.PHONY: metricsdocs build-metricsdocs
+
+metricsdocs: build-metricsdocs
+	@[ "${GITHUB_TOKEN_FILE}" ] || ( echo "GITHUB_TOKEN_FILE is not set"; exit 1 )
+	@[ "${CONFIG_FILE}" ] || ( echo "CONFIG_FILE is not set"; exit 1 )
+
+	tools/metricsdocs/_out/metricsdocs \
+		--github-token-file $(GITHUB_TOKEN_FILE) \
```

Comment:
Why do you need to supply the github token if the tool is just cloning upstream repositories?
Can't it be done without a token?

Reply:
yes totally, removed